source/retryable-writes/retryable-writes.rst

```diff
@@ -10,7 +10,7 @@ Retryable Writes
 :Status: Accepted
 :Type: Standards
 :Minimum Server Version: 3.6
-:Last Modified: 2019-10-21
+:Last Modified: 2020-01-14
```

Comment:
Since we're introducing a new error code, I think the spec version can also be bumped up a minor version to 1.5.0.